guides/source/configuring.md

```diff
@@ -819,9 +819,7 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.time_precision` sets the precision of JSON encoded time values. Defaults to `3`.
 
-* `config.active_support.use_sha1_digests` specifies whether to use SHA-1 instead of MD5 to generate non-sensitive digests, such as the ETag header.
```

Comment:
Are deprecated things dropped from the docs or should they also have a deprecation warning here?

Reply:
Drop from the docs, but it needs to be mentioned in the `railties/CHANGELOG.md` file.

Reply:
@eileencodes I've added a changelog entry as well.